### PR TITLE
Bugfix/naming consistency

### PR DESCRIPTION
**Pull request by jokedurnez**
Found this a bit a weird function name with feedbacks when returning an iterable and feedback when it's singular. It's grammatically incorrect.

### CODE REVIEW

File: src/main/java/com/openlattice/hazelcast/HazelcastMap.java [hunk 1]
```diff
@@ -67,7 +67,7 @@
     LINKING_EDGES,
     LINKING_ENTITIES,
     LINKING_ENTITY_VERTICES,
-    LINKING_FEEDBACKS,
+    LINKING_FEEDBACK,
```
Comment (katherinebernstein) [inline src/main/java/com/openlattice/hazelcast/HazelcastMap.java]: Is there a mpastore for this? If there is the name needs to be updated there too

Comment (you-leee) [reply src/main/java/com/openlattice/hazelcast/HazelcastMap.java]: +1

Comment (you-leee) [reply src/main/java/com/openlattice/hazelcast/HazelcastMap.java]: `LinkingFeedbackMapstore`

Comment (jokedurnez) [reply src/main/java/com/openlattice/hazelcast/HazelcastMap.java]: ok done... I think...  Is the commit below enough?

Comment (katherinebernstein) [reply src/main/java/com/openlattice/hazelcast/HazelcastMap.java]: If you're using intellij to do the refactor it should update all usages for you so you don't have to search through all the projects or worry about missing any (and you can also find all usages using intellij to double check)

**Pull request merged**